tests/test_back.js

```diff
@@ -151,30 +151,26 @@ tap.test('nockBack dryrun tests', function (nw) {
   nockBack.fixtures = __dirname + '/fixtures';
   nockBack.setMode('dryrun');
 
-  nw.test('goes to internet even when no nockBacks are running', {skip: process.env.AIRPLANE}, function(t) {
-    var req = http.request({
-        host: "www.amazon.com"
-      , path: '/'
-      , port: 80
-      }, function(res) {
-
-        res.on('data', function() {
-          //node v 0.10 requires this listener
-        });
-        t.ok([200, 301, 302].indexOf(res.statusCode) >= 0);
-        t.end();
-
-      });
-
-    req.on('error', function(err) {
-
-      //  This should never happen.
-      t.assert(false);
-      t.end();
-
-    });
-
-    req.end();
+  nw.test('goes to internet even when no nockBacks are running', function(t) {
+    t.plan(2)
+
+    const server = http.createServer((request, response) => {
+      t.pass('server receives request')
+      response.writeHead(200)
+      response.end()
+    })
+    server.listen(() => {
```

Comment:
Note that `server.listen(() => {...})` will automatically get an available port assigned, so tests can run in parallel without conflicts, see [`server.listen([port][, host][, backlog][, callback])`](https://nodejs.org/api/net.html#net_server_listen_port_host_backlog_callback)

Reply:
Oh btw, you need the semi-colons ;)

Reply:
lol yeah we can put in proper syntax rules once we addressed the existing issues & PRs. I’ll add prettier, but not yet. For now it's wild wild west I don’t care